<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>enterWindow</class>
 <widget class="QWidget" name="enterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>enterWindow</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/paper-background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="introText_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>521</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>32</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    border: none;
    background: none;
   color: rgb(123, 156, 100);
}</string>
   </property>
   <property name="text">
    <string>Введите имя пользователя и пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordField">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Light</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
                   font-size: 10px; 
                   background-color: rgb(123, 156, 100);
                   border: 1px solid gray; 
                   border-radius: 10px; 
                   padding: 5px; 
                }
QPushButton:hover {
    background-color: rgb(255, 174, 205); 
    cursor: pointer; /* Изменение курсора при наведении */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginField">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  background-color: rgb(123, 156, 100);
            border: 1px solid gray; 
               border-radius: 10px; 
               padding: 5px; 
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(123, 156, 100);
    color: white; /* Цвет текста */
    border: none; /* Без рамки */
    border-radius: 10px; /* Закругленные границы */
    padding: 10px 20px; /* Отступы внутри кнопки */
    font-size: 16px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: rgb(255, 174, 205); 
    cursor: pointer; /* Изменение курсора при наведении */
}</string>
   </property>
   <property name="text">
    <string>Подтведить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(123, 156, 100);
    color: white; /* Цвет текста */
    border: none; /* Без рамки */
    border-radius: 10px; /* Закругленные границы */
    padding: 10px 20px; /* Отступы внутри кнопки */
    font-size: 16px; /* Размер шрифта */
}

QPushButton:hover {
    background-color: rgb(255, 174, 205); 
    cursor: pointer; /* Изменение курсора при наведении */
}</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="background_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Qlabel {
    border: none;
    background: none;
    border-radius: 15px; 
    background-color: transparent
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../resources/sun.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
